### PR TITLE
fix : lib/include may be file after install

### PR DESCRIPTION
**Pull request by chanchann**
https://github.com/xmake-io/xmake-repo/pull/1750

### CODE REVIEW

File: src/xmake.lua [hunk 1]
```diff
@@ -38,6 +38,8 @@ target("workflow")
     end)
 
     on_install(function (target)
+	os.mkdir(target:installdir("include"))
+	os.mkdir(target:installdir("lib"))
```
Comment (waruqi) [inline src/xmake.lua:42]: 感觉缩进有问题

Comment (chanchann) [reply src/xmake.lua:42]: 刚才重新提了个，https://github.com/sogou/workflow/pull/1154

Comment (chanchann) [reply src/xmake.lua:42]: > 另外，target:installdir 目前是不带参数的，只有 package 能这么写，这里得改成
> 
> ```lua
> 	os.mkdir(path.join(target:installdir(), "include"))
> ```

ok, 好的大佬

Comment (chanchann) [reply src/xmake.lua:42]: https://github.com/sogou/workflow/pull/1155

**Pull request merged**